install.sh

```diff
@@ -64,26 +64,39 @@ function clone_repo() {
 }
 
 function unjoin() {
+  for id in $(kubectl get foreignclusters -o jsonpath="{.items[*].metadata.name}"); do
+    kubectl patch foreignclusters "$id" -p '{"spec":{"join":false}}' --type 'merge'
+  done
+
+  for id in $(kubectl get advertisements -o jsonpath="{.items[*].metadata.name}"); do
+    kubectl patch advertisements "$id" -p '{"status":{"advertisementStatus":"Deleting"}}' --type 'merge'
+  done
```

Comment:
My 2 cents: This second patch should not be necessary, since the first one should imply the status change of advertisements